install/0000_30_machine-config-operator-00_clusterreader_clusterrole.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:openshift:machine-config-operator:cluster-reader
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-cluster-reader: "true"
+rules:
+  - apiGroups:
+      - machineconfiguration.openshift.io
+    resources:
+      - containerruntimeconfigs
+      - controllerconfigs
+      - kubeletconfigs
+      - machineconfigpools
+      - mcoconfigs
```

Comment:
The machineconfig objects can end up with secrets embedded though, like the pull secret.

Reply:
That's not the machineconfigs tho. I believe at this point it's just a leftover from the early days of this operator and currently contains nothing

Reply:
are you both talking about the `mcoconfigs`?

Reply:
Yep, this was discussed in Slack too I see now, and I was indeed thinking `mcoconfigs` was `machineconfigs`.